subsys/partition_manager/Kconfig

```diff
@@ -15,6 +15,11 @@ config PARTITION_MANAGER_ENABLED
 	  in how parent images impose options on child images. Don't
 	  change the value of this option.
 
+config FLASH_MAP_CUSTOM
```

Comment:
Maybe we should add a comment here stating why we override this option? Something along the lines of "Override this option to use custom flash map implementation when partition manager is enabled."